src/core/bpf-foreign.c

```diff
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: LGPL-2.1+ */
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
```

Comment:
Wow, how did we miss those in the previous conversions?

Reply:
I think these are new files, it was added like this once and then probably copied and pasted all over